foundry/foundry.py

```diff
@@ -199,11 +200,10 @@ def load(self, name, download=True, globus=True, verbose=False, metadata=None, a
 
         # TODO: Creating a new Foundry instance is a problematic way to update the metadata,
         # we should find a way to abstract this.
-
-        fdataset = FoundryDataset(**res['dataset'])
+        
         self.dc = res['dc']
         self.mdf = res['mdf']
-        self.dataset = fdataset
+        self.dataset = FoundryDataset(**res['dataset'])
 
 
         if download is True:  # Add check for package existence
```

Comment:
Sorry man, another thing caught my attention? Why `is True`? Why not just `if dataset`?

Reply:
oh I have no clue, I'll fix that though.